### PR TITLE
Hasura migrations

### PR DESCRIPTION
**Pull request by mddilley**
## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/13293

This PR removes the previous manual tracking of Hasura metadata, schema, and migrations. It adds the files generated by [hasura init](https://hasura.io/docs/latest/hasura-cli/commands/hasura_init/). You can see the steps of what I did [here](https://github.com/cityofaustin/atd-data-tech/issues/13293#issuecomment-1692609571).

**Forgot to mention that a ton of these files are autogenerated so I don't think a line by line review is needed. I added some comments, and I think that it really helps to look at the files/structure in a code editor.**

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Checkout this branch, start the virtual environment, and run `./vision-zero replicate-db`
2. Run `./vision-zero hasura-console` to open the console and make the changes in the next steps
3. In the console, go to **Data** > **default** > **public** > **atd_txdot_crashes**
4. Then, go to **Modify** > **Add a new column** to create a column with:
   - column name: **new_column**
   - type: **text**
   - no default value
   - nullable
5. Now, go to the **Permissions** tab and you should see that the **editor** and **readonly** have two cells in the permissions grid with the filter icon
6. Click those filters icons and click the checkbox next to **new_column** under **Column select permissions** and save
7. In your code editor, you should see:
    - `/atd-vz-data/atd-vzd/metadata/databases/default/tables/public_atd_txdot_crashes.yaml` now shows a diff with select permissions for both **editor** and **readonly** roles
    - there is a new migration folder in `/atd-vz-data/atd-vzd/migrations/default` with autogenerated up and down migration files for the new column
8. These would be the changes that we would review in PRs and then apply using the Hasura CLI when merging to `master` and eventually releasing to production. I can write up some docs on that before closing this issue out.

---
#### Ship list
- [x] Code reviewed
- [x] Product manager approved

### CODE REVIEW

File: atd-vzd/config.yaml
Comment (mddilley) [inline atd-vzd/config.yaml:1]: this file sets up what endpoint and secret is used when running `hasura console`

File: atd-vzd/hasura/hasura_metadata_2023_04_11_14_03_58_508.json
Comment (mddilley) [inline atd-vzd/hasura/hasura_metadata_2023_04_11_14_03_58_508.json:1]: old Hasura metadata tracking that we added manually 👋

File: atd-vzd/migrations/default/1692924316671_init/up.sql
Comment (mddilley) [inline atd-vzd/migrations/default/1692924316671_init/up.sql:1]: the giant migration file 🐘🐘 with the current state of things

File: atd-vzd/migrations/migration_atd_txdot__veh_unit_desc_lkp_and_atd_txdot__veh_body_styl_lkp_2023-05-11--1464.sql
Comment (mddilley) [inline atd-vzd/migrations/migration_atd_txdot__veh_unit_desc_lkp_and_atd_txdot__veh_body_styl_lkp_2023-05-11--1464.sql:1]: all files like this removed from its folder were the previous migration-like files that we used to track changes

File: atd-vzd/schema/afd__incidents.sql
Comment (mddilley) [inline atd-vzd/schema/afd__incidents.sql:1]: same here, this was our previous way of tracking the schema

File: vision-zero [hunk 1]
```diff
@@ -249,6 +251,15 @@ def graphqlEngineUp(args):
...
     subprocess.run(["docker", "ps"])
+    print("🚀  GraphQL Engine started.")
+    print("You can run './vision-zero hasura-console' to open the console.")
+
+def hasuraConsole(args):
+    print("🚀  Opening Hasura console")
+    try:
+        subprocess.call(["hasura", "console"], cwd="./atd-vzd")
+    except KeyboardInterrupt:
+        print('Exited Hasura console')
```
Comment (mddilley) [inline vision-zero:262]: Open to thoughts on this, my thinking was:
1. we run `./vision-zero replicate-db` to get the local db and Hasura running
2. it prompts what command to run next (like the Moped `hasura-cluster` does) to get the console running so we can generate schema changes and migrations through it
3. it was either `./vision-zero hasura-console` or `cd atd-vzd` and then `hasura console`
Also, I tried just starting the console as the last step of `./vision-zero replicate-db` but it errors if Hasura takes a moment to spin up and also locks up the terminal. Didn't want to go down that rabbit hole.

**Review by roseeichelmann — APPROVED** [review]
test steps checked out, up/down migrations files were auto generated for me and the permissions metadata were updated! thanks so much for doing this mike, i am so excited to have this kind of automation in VZD, its gonna be a game changer!! :partying_face:

**Review by frankhereford — APPROVED** [review]
I love how this splits up all the metadata! 😍 

I've looked at this, and it looks just exactly like what I would expect to be in here. The changes to the `vision-zero` orchestration program are great too.

This is 💯 🎯! Thank you @mddilley!

**Review by patrickm02L — APPROVED** [review]
Adding my concurrence based on Dev reviews 💎

**Pull request merged**